pkg/registry/thirdpartyresource/strategy.go

```diff
@@ -48,6 +47,10 @@ func (strategy) NamespaceScoped() bool {
 	return false
 }
 
+func (strategy) GenerateName(base string) string {
```

Comment:
We have to implement `GenerateName` to satisfy `RESTCreateStrategy`, but I really hope someone doesn't make use of generated names to determine their custom api groups and kinds. Not really sure what to generate here, open to ideas